package.json

```diff
@@ -7,7 +7,7 @@
 		"packages/*"
 	],
 	"scripts": {
-		"build:all": "nx run-many --target=build",
+		"build:all": "nx run-many --target=build && npx typedoc",
```

Comment:
do we have to build all packages for the typedoc? if not i think we can make this command separate

Reply:
Nope, just proto -> core, react. Updated with a new docs script that builds what is required for docs and the docs themselves.